mpifileutils.spec

```diff
@@ -249,6 +248,12 @@ done
 %endif
 
 %changelog
+* Mon Feb 07 2022 Dalton A. Bohning <daltonx.bohning@intel.com> - 0.11.1-1
+- Update to v0.11.1
+- Remove patch2 in favor of upstream d5f41b3
```

Comment:
Is d5f41b3 0.11.1?  If so let's change this reference to the version number.

Reply:
d5f41b3 is a specific commit that landed upstream. patch2 was temporary until d5f41b3 was merged properly. I.e. the contents of patch2 and d5f41b3 are the same